javascript-timers/interval.js

```diff
@@ -0,0 +1,20 @@
+var heading = document.querySelector('h1');
+var counter = 4;
+
+function countdown() {
+  heading.textContent = counter;
+  if (counter === 4) {
+    counter--;
+  } else if (counter === 3) {
+    counter--;
+  } else if (counter === 2) {
+    counter--;
+  } else if (counter === 1) {
+    counter--;
```

Comment:
It would be better to do a single check here, if `counter` is greater than 0, do something, else do something else.